src/index.ts

```diff
@@ -1,4 +1,4 @@
-import dotenv from "dotenv";
+import * as dotenv from "dotenv";
```

Comment:
As a codeguard, here are my suggestions:

1. Line 5: It is a good practice to put the `dotenv.config()` call in a try-catch block to handle the case where the environment file is not found.
```ts
try {
  dotenv.config();
} catch (error) {
  console.error("Error loading environment variables:", error);
}
```

2. Line 12: The `cookie` header should be set to an empty string instead of `process.env.CHATGPT_COOKIES || ""` to avoid leaking sensitive information.
```ts
"cookie": ""
```

3. Line 13: The `Authorization` header should be set to an empty string instead of `process.env.CHATGPT_AUTH_TOEKN || ""` to avoid leaking sensitive information.
```ts
"Authorization": ""
```

4. Line 28: It is a good practice to add type definitions for the parameters in the function signature, to make it easier for others to understand the inputs and outputs of the function.
```ts
export async function sendPostRequest(
  options: SendPostRequestOptions = {}
): Promise<ChatGPTResponse> {
```

5. Line 40: It is a good practice to add a `timeout` option to the `fetch` call to prevent the function from hanging indefinitely.
```ts
const response = await fetch(API_ENDPOINT, {
  method: "POST",
  headers: HEADERS,
  body: JSON.stringify({
    ...
  }),
  timeout: 5000,
});
```

6. Line 79: It is a good practice to throw a more descriptive error message instead of just the error object.
```ts
throw new Error(`Error parsing response from ChatGPT API: ${error.message}`);
```

7. Line 82: It is a good practice to throw a more descriptive error message instead of just the error object.
```ts
throw new Error(`Error sending POST request to ChatGPT API: ${error.message}`);
```